### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by halildurmus**
I've transferred the repo to my personal account, so updating the link accordingly.

### CODE REVIEW

**Comment by 0x5bfa** [comment]
@halildurmus the link exists also in [projections.md](https://github.com/microsoft/win32metadata/blob/main/docs/projections.md)

**Comment by halildurmus** [comment]
@microsoft-github-policy-service agree

**Review by mikebattista — APPROVED** [review]

**Comment by riverar** [comment]
And who are you @halildurmus? Do you work at Google / work on Dart officially?

**Comment by kennykerr** [comment]
I thought this was @timsneath's project...

**Comment by timsneath** [comment]
I can vouch for @halildurmus as the legitimate successor owner of this project :) 

I left Google a while back and am now working on... Swift-ier things at Apple. Halil was by far the largest contributor to the Win32 Dart implementation, and he graciously agreed to take over stewardship when I left. The Win32 project is endorsed by the Flutter team as a "Flutter Favorite" and is extensively used as a core part of the Windows support in Flutter. 

In short, this is legit; hope this helps! 

_I'd work on the Swift version of this package, but fortunately Swift builds on top of LLVM so can natively call into Win32 libraries without additional ceremony :)_

**Comment by riverar** [comment]
Ah cool, thanks and welcome!
